website/docs/components/icon-tile/partials/guidelines/guidelines.md

```diff
@@ -58,4 +58,20 @@ A secondary icon can be added to provide additional context. For example, for an
 
 ## Using with content
 
-Icon Tiles should not appear without accompanying content. Instead, they should be presented alongside a text label for the object or page they are being used to represent. We recommend thinking of them as an accessory.
\ No newline at end of file
+Icon Tiles should not appear without accompanying content. Instead, they should be presented alongside a text label for the object or page they are being used to represent. We recommend thinking of them as an accessory.
+
+## Near interactive elements
+
+Avoid placing Icon Tiles next to buttons or other clickable elements. This can confuse users and make it unclear what is interactive. In these cases, use a simple icon for better usability.
```

Comment:
I tried rewording this so that it can be a little more descriptive (plus providing a source) as to why we want folks to be mindful of this.

```suggestion
Avoid placing Icon Tiles next to visually similar interactive elements. [Distinguishability](https://www.w3.org/WAI/WCAG21/Understanding/distinguishable) is an accessibility standard that deters interfaces from using elements that look the same but behave differently in close visual proximity. This can cause user confusion through the established visual language. 
```